### DISCUSSION TITLE
When is the release 2.5.1 planned

### DISCUSSION THREAD

**Discussion by Amitkehri** [iBotPeaches/Apktool#2540] [Q&A] [answered]
Hi Team ApkTool/ @iBotPeaches ,

Any tentative idea when version 2.5.1 will be released ?

Please let me know.

Thanks,
Amit.

**Comment by iBotPeaches** [comment, marked as answer]
Towards end of May is the goal - https://github.com/iBotPeaches/Apktool/milestone/20
